sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/Main.java

```diff
@@ -142,21 +148,27 @@ void runAll(String[] args) throws IOException {
         saveSummary(null, configurations, actual, baseline, start, options);
       }
 
-      if (options.getExportSummaryToBigQuery()) {
-        ImmutableMap<String, String> schema =
-            ImmutableMap.<String, String>builder()
-                .put("timestamp", "timestamp")
-                .put("runtimeSec", "float")
-                .put("eventsPerSec", "float")
-                .put("numResults", "integer")
-                .build();
+      final ImmutableMap<String, String> schema =
+          ImmutableMap.<String, String>builder()
+              .put("timestamp", "timestamp")
+              .put("runtimeSec", "float")
+              .put("eventsPerSec", "float")
+              .put("numResults", "integer")
+              .build();
 
+      if (options.getExportSummaryToBigQuery()) {
         savePerfsToBigQuery(
             BigQueryResultsPublisher.create(options.getBigQueryDataset(), schema),
             options,
             actual,
             start);
       }
+
+      if (options.getExportSummaryToInfluxDB()) {
+        final long timestamp = start.getMillis() / 1000; // seconds
```

Comment:
The default precision is nanoseconds. In case of nexmark results we changed it and use seconds instead
```
return new HttpPost(
        settings.host + "/write?db=" + settings.database + "&" + retentionPolicy + "&precision=s");
```

Reply:
We've thought that we don't really need milliseconds. Even seconds are probably more than enough

Reply:
Yes you guys are right seconds is ok for the timestamps but ms are good for the benchmark results